opsdroid/events.py

```diff
@@ -478,3 +482,14 @@ class PinMessage(Event):
 
 class UnpinMessage(Event):
     """Event to represent unpinning a message or other event."""
+
+
+class DeleteMessage(Event):
+    """Event to represent deleting a message or other event."""
+
+    def __init__(self, id, *args, **kwargs):  # noqa: D107
```

Comment:
I think `id` should just be `linked_event`?

Reply:
sounds good, in the Twitch case, we need to pass the message id to delete it.  I'm assuming that the linked_event represents the whole Message event right?